modules/@apostrophecms/area/ui/apos/components/AposAreaExpandedMenu.vue

```diff
@@ -220,6 +220,12 @@ export default {
     background-color: var(--a-base-10);
   }
 
+  &--1-column {
+    display: grid;
+    grid-template-columns: repeat(1, 1fr);
```

Comment:
```suggestion
    grid-template-columns: 1fr;
```
No real need for the `repeat`, this is cleaner.